extensions/clusters/clusters.go

```diff
@@ -271,6 +271,7 @@ func NewRKE1ClusterConfig(clusterName string, client *rancher.Client, clustersCo
 	}
 	newConfig.ClusterAgentDeploymentCustomization = clustersConfig.ClusterAgent
 	newConfig.FleetAgentDeploymentCustomization = clustersConfig.FleetAgent
+	newConfig.AgentEnvVars = clustersConfig.AgentEnvVarsRKE1
```

Comment:
I think you need to choose between having this and the one encapsulated in the 'if' statement below